### ISSUE TITLE
Extension and content of file validators.fi.yml don't match

### ISSUE THREAD

**Issue by alejandro-arrabal** [misd-service-development/phone-number-bundle#100]
The content format is xliff and the extension is yaml, wich trigger this exception:
```

[Symfony\Component\Translation\Exception\InvalidResourceException]                                                                
  Error parsing YAML, invalid file "<rootPath>/vendor/misd/phone-number-bundle/Resources/translations/validators.fi.yml"  

```
related with merged pull request  #93

**Comment by peshi** [comment]
The same is true for https://github.com/misd-service-development/phone-number-bundle/pull/94

**Comment by lunetics** [comment]
Someone up for fixing?

**Comment by peshi** [comment]
Fix: https://github.com/misd-service-development/phone-number-bundle/pull/101

**Comment by thewilkybarkid** [comment]
Merged and 1.1.3 released.

I thought I'd already released it, but hadn't actually pushed it! So, thanks for catching this.
